aspnetcore/includes/managed-identities-test-non-production.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> Microsoft recommends that you use the most secure authentication flow available. The authentication flow described in this procedure requires a very high degree of trust in the application, and carries risks that are not present in other flows. You should only use this flow when other more secure flows, such as managed identities, aren't viable.
```

Comment:
```suggestion
<!-- we know what the most secure is, so make it a link and don't be so nebuluous and send them off hunting for the solution. -->
> Microsoft recommends that you use the [most secure authentication flow available](appropriate URL). The authentication flow described in this procedure:

* Is only suitable for test apps.
* Requires a very high degree of trust in the app.
* Carries risks that are not present in other flows.

This authentication flow should only be used in test apps. For production apps, use the most secure authentication flow available. <!-- TODO, links to recommended authentication -->
```

Reply:
Thanks @Rick-Anderson! Sorry, I don't know what the most secure authentication flow available is that we are now recommending now for this situation that I would link to.  We are assuming either SQL server or SQLite in this tutorial and more than one environment, but not assuming Azure nor an Azure account.

Reply:
> Thanks @Rick-Anderson! Sorry, I don't know what the most secure authentication flow available is that we are recommending now for this situation that I would link to. Using this tutorial I am linking it to as an example, we are assuming either SQL server or SQLite in this tutorial and more than one environment, but not assuming Azure nor an Azure account.

My suggestion fixed